python/ray/tune/tune.py

```diff
@@ -953,6 +952,9 @@ class and registered trainables.
 
     progress_metrics = _detect_progress_metrics(_get_trainable(run_or_experiment))
 
+    if _use_storage_context():
+        air_usage.tag_storage_type(experiments[0].storage)
+
```

Comment:
How come this is tracked down here rather than where the TODO was?

Reply:
I need the `Experiment` object to be initialized to access the `StorageContext`